docker/README.md

```diff
@@ -0,0 +1,104 @@
+Miaou development environment using Docker
+===
+**Build a local tiny Virtual Machine which allows you to run Miaou on any OS (including Windows/OS X)**
```

Comment:
I'm not a fan of the whole docker-machine thingie. Not that I don't like it, but I'd rather offer 2 ways: using docker on its own (for people on linux), or using docker machine. In this document, it looks like docker machine is the only way.